Sources/ClientRuntime/Networking/Http/Middlewares/SignerMiddleware.swift

```diff
@@ -50,8 +50,14 @@ extension SignerMiddleware: ApplySigner {
             )
         }
 
-        // Check if CRT should be provided a pre-computed Sha256 SignedBodyValue
         var updatedSigningProperties = signingProperties
+
```

Comment:
Below, the clock skew store is checked for a clock skew value. If one exists, it is entered into signing properties. Then it will be applied when calculating the signature.